Amplifier_Operational.dcm

```diff
@@ -180,6 +180,18 @@ K single jfet opamp
 F http://www.ti.com/lit/ds/symlink/lf357.pdf
 $ENDCMP
 #
+$CMP LF351D
+D Wide bandwidth single JFET operational amplifier, SOIC-8
+K Singel, OpAmp, JFET
```

Comment:
typo. 'singel' vs. 'single'
Also commas don't belong into keywords.
But description should have commas.